_applications/fabric8-maven-plugin.md

```diff
@@ -0,0 +1,95 @@
+- https://github.com/nicolaferraro/voxxed-bigdata-kafka
+- https://github.com/nicolaferraro/voxxed-bigdata-web
+- https://github.com/nicolaferraro/voxxed-bigdata-spark
+- https://github.com/nicolaferraro/voxxed-bigdata-pipeline
+---
+
+<h1 id="introduction">Introduction</h1>
+
+This demo shows how to use the Fabric8 Maven Plugin to deploy a Spark cluster on Openshift.
+
+The application simulates a movie delivery website (like Youtube, or Netflix) where users can 
+  rate movies and get recommendations from the system.
+  Recommendations are computed by a backend Apache Spark cluster running a basic collaborative 
+  filtering algorithm called ["slope one"](https://en.wikipedia.org/wiki/Slope_One).
+
+<h1 id="architecture">Architecture</h1>
+
+A high level overview of the final system is summarized in the following schema:
+
+<img src="/assets/fabric8-maven-plugin/architecture.png" class="img-responsive">
+
+The system is composed of:
+
+- A Spring-Boot front-end application: delivering static content and exposing rest services through which 
+ the UI can send ratings and lookup recommendations. [Apache Camel](https://camel.apache.org) is used for all integration purposes.
+- A Kafka broker with two topics: "stars" containing all ratings from the users; "recommendations" containing recommendations sent by the Spark application.
+- A Oshinko Spark cluster: it computes the recommendations and pushes them back to Kafka.
+
+This is a demo application, so components are not intended for production usage. E.g. the Kafka broker (containing also Zookeeper) 
+cannot be scaled out. The Spark application computes recommendations in a window of 10 minutes for simplicity.
+The web application is using a local cache to store recommendations received from Kafka, so only one instance should be running.
+
+<h1 id="installation">Installation</h1>
+
+The demo requires a Openshift installation.
+All three components are hosted in different git repositories and can be installed on Openshift using the [Fabric8 Maven Plugin](https://maven.fabric8.io/).
+
+The first step is to deploy the Kafka broker. You can clone the repository at the following [link](https://github.com/nicolaferraro/voxxed-bigdata-kafka), 
+then execute the following command from the project root:
+
+```
+mvn clean fabric8:deploy
+```
+
+The Fabric8 Maven Plugin will use the current Openshift session to access Openshift. You're automatically logged in when you create a new cluster, 
+or you can use the command `oc login` to authenticate again.
+
+The second step is to deploy the web application. You can clone the repository at the following [link](https://github.com/nicolaferraro/voxxed-bigdata-web) and run 
+a `mvn clean fabric8:deploy`.
+
+Finally, you need to deploy the Spark cluster. First, you need to create the 
+Oshinko resources (service account and roles) in the current Openshift project, through the following command:
+
+```
+oc create -f http://radanalytics.io/resources.yaml
+```
```

Comment:
our preference has been to link back to the "Get Started" page instead of recreating the instructions in the tutorials